src/billing/api/index.ts

```diff
@@ -8,16 +8,26 @@ import {
   Account,
 } from 'src/client/unityRoutes'
 
-import {RemoteDataState} from 'src/types'
+import {
+  RemoteDataState,
+  PaymentInformation,
+  PostAccountUpgradeResult,
+} from 'src/types'
 import {
   Invoice,
   CreditCardParams,
   Region,
   BillingNotifySettings,
   PaymentMethod,
+  ZuoraParams,
 } from 'src/types/billing'
 
-const makeResponse = (status, data) => {
+const makeResponse = (status, data, respName, ...args) => {
+  console.log(respName) // eslint-disable-line no-console
```

Comment:
uhhhhh why?

Reply:
It was in src/unity/api/index.ts makeResponse. So, I kept it. Removing.

Reply:
thanks!